src/index.tsx

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '@app/components/app/app.component';
+import 'index.css'; // TODO: import styles directly into index.html using webpack
```

Comment:
Since you're using css in js approach, let's put global styles to jss as well and get rid of those css files

Reply:
It turned out to be not that simple :)
I didn't find a solution how to specify some pseudo-selectors (like ::selection) with jss approach, as well as import the normalize.css package.
I noticed that I spent more time than I expected trying to solve a minor issue with importing everything via jss and decided to leave it as it is. I hope it isn't a big deal to have several .css files in the project :)

In general I would say jss lib isn't as friendly as it might seem at first glance, even using jss with React class components requires additional webpack settings and plugins. Quite a lot of work for a small styles library)

Reply:
okay, we can leave it as it is. The only benefit of having this in jss is possibility of using theme, but since you don't have one it's not a problem :)